slugid/slugid.py

```diff
@@ -1,8 +1,10 @@
 # Licensed under the Mozilla Public Licence 2.0.
 # https://www.mozilla.org/en-US/MPL/2.0
 
+import sys
 import uuid
 import base64
+two = True if sys.version_info.major == 2 else False
```

Comment:
Can we rename `two` to e.g. `isPython2`?

Alternatively as `sys.version_info.major` already holds this information, we could directly evaluate on line 21, e.g.

```
if sys.version_info.major == 2 and isinstance(slug, bytes):
```

Performance-wise I'm guessing the two options above are similar. I find the second option more readable than introducing an extra variable, (e.g. if python 4 came out, we'd potentially need `isPython3` and `isPython4` variables, whereas directly evaluating a condition against `sys.version_info.major` means we can already test if it is less/equal/bigger than a given value or satisfies a certain range etc. on a case-per-case basis).